orphans/src/Orphans.hs

```diff
@@ -0,0 +1,18 @@
+{-# LANGUAGE DeriveAnyClass     #-}
+{-# LANGUAGE FlexibleInstances  #-}
+{-# LANGUAGE OverloadedStrings  #-}
+{-# LANGUAGE StandaloneDeriving #-}
+module Orphans () where
```

Comment:
I'm not sure if creating a package just for orphans is enough. Maybe we can call something like `plutus-extra` and put Orphans module there?

I also think that maybe we should add to the naming some information what orphans are for. For example here we can call it `Orpahns.OpenApi` or something.